sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationAsyncClientTest.java

```diff
@@ -706,15 +706,15 @@ public void listRevisionsWithPagination() {
 
         List<Mono<Response<ConfigurationSetting>>> results = new ArrayList<>();
         for (ConfigurationSetting setting : settings) {
-            results.add(client.setSetting(setting));
```

Comment:
Are these changes to prevent a NPE from Reactor or is it just that the non-WithResponse doesn't trigger the onNext or onError handler?

Reply:
No, this wasn't for either of the cases. It was actually to simply to comply with `Mono<Response<T>>` return type.

But [here](https://github.com/Azure/azure-sdk-for-java/pull/4450/files#diff-6496d4e3f2fd43c5f16c599ecfb1da47R115) is to deal with NPE from Reactor and sending a Mono.Empty on null response.